upgrade.py

```diff
@@ -301,3 +301,12 @@ def run_upgrades():
         if table_exists("command_config"):
             db.exec("UPDATE command_config SET access_level = 'superadmin' WHERE command = 'system'")
         version = update_version(version)
+
+    if version == 33:
+        if table_exists("points_log"):
+            db.exec("ALTER TABLE points_log RENAME TO points_log_old")
+            db.exec("CREATE TABLE IF NOT EXISTS points_log (log_id INT PRIMARY KEY AUTO_INCREMENT, char_id BIGINT NOT NULL, audit INT NOT NULL, leader_id BIGINT NOT NULL, reason VARCHAR(255), created_at INT NOT NULL)")
+            db.exec("INSERT INTO points_log (char_id, audit, leader_id, reason, created_at) SELECT char_id, audit, leader_id, reason, created_at FROM points_log_old ORDER BY created_at ASC")
```

Comment:
@bigwheels16 I'm not sure, if its okay for the log_ids to change, the order by should keep them in the same order, unless someone modified their row's `created_at` by hand.

Reply:
We need to include the log_id when we copy the values from the old table to the new in the list of fields to prevent the log_ids from changing.

Once you add that I think it should be good.